source/features/expand-urls.js

```diff
@@ -1,16 +1,52 @@
+import ky from 'ky';
+
 const refinedTwitterClass = 'refinedTwitterExpandedUrl';
 
+async function getUnshortenUrl(shortURL) {
+	console.log("getUnshortenUrl " + shortURL);
+
+	const json = await ky.get('https://linkpeelr.appspot.com/api?action=peel_all&url=' + shortURL + '&where=twitter.com&version=2.0.3',
```

Comment:
it always return me an empty answer whereas I can see a full payload on my network tab on Chrome, can you give me a hint of this please @sindresorhus ?

```
body: (...)
bodyUsed: false
headers: Headers {}
ok: false
redirected: false
status: 0
statusText: ""
type: "opaque"
url: ""
```

Reply:
```js
ky.get(url).json()
```
 ?

Reply:
So, when using a `mode: 'no-cors'` with a `fetch()` we get a opaque response with status:0
https://fetch.spec.whatwg.org/#concept-filtered-response-
https://fullstack-developer.academy/what-is-an-opaque-response/. 
The problem does not come from ky :heavy_check_mark: 

I don't know why I need to use `mode: 'no-cors'` with fetch() where I'm not using it with [XMLHttpRequest](https://github.com/polomarcus/faster-links/blob/910eeaf7d78c83d47dbf9116fde143ccf5c77480/app/scripts.babel/contentscript.js#L84-L97)

Reply:
Alright, go for XMLHttpRequest then.